js/ui/date_box/ui.date_box.base.js

```diff
@@ -407,42 +398,32 @@ var DateBox = DropDownEditor.inherit({
         $element.addClass(DATEBOX_CLASS + "-" + this._pickerType);
     },
 
-    _updateSize: function() {
-        var $element = this.$element(),
-            widthOption = this.option("width"),
-            isWidthSet = typeUtils.isDefined(widthOption) || (isRealWidthSet($element) && !this._isSizeUpdatable),
-            pickerType = this._pickerType,
-            // NOTE: no calculateWidth if type is rollers, why?
-            shouldCalculateWidth = pickerType !== PICKER_TYPE.rollers && devices.current().platform === "generic";
-
-        if(!windowUtils.hasWindow() || isWidthSet || !(shouldCalculateWidth && $element.is(":visible"))) {
-            return;
-        }
+    _formatValidationIcon: function(value) {
+        const longestValue = "12-12-1900";
```

Comment:
Why 12-12-1900 is the longest value?

Reply:
Fixed: longest value calculates automatically now.